lib/apartment/adapters/postgresql_adapter.rb

```diff
@@ -155,7 +155,7 @@ def pg_dump_schema
       #   @return {String} raw SQL contaning inserts with data from schema_migrations
       #
       def pg_dump_schema_migrations_data
-        with_pg_env { `pg_dump -a --inserts -t schema_migrations -t ar_internal_metadata -n #{default_tenant} #{dbname}` }
```

Comment:
Is there any reason to remove this?

Reply:
It was failing in my local application. I looked more closely, and found a reference in the [pg_dump docs](https://www.postgresql.org/docs/10/static/app-pgdump.html)

> The -n and -N switches have no effect when -t is used, because tables selected by -t will be dumped regardless of those switches, and non-table objects will not be dumped.

So I'm explicitly namespacing the tables instead.